ocrd_network/ocrd_network/processing_server.py

```diff
@@ -735,3 +759,21 @@ async def run_workflow(
         )
         await db_workflow_job.insert()
         return db_workflow_job.to_job_output()
```

Comment:
For suggestion 1:

```suggestion
        await db_workflow_job.insert()
        # Do not return the processing_job_ids to the user
        db_workflow_job[processing_job_ids] = {}
        return db_workflow_job.to_job_output()
```

Reply:
I committed first and will check later/now, and I am going to resolve conversation then.

Reply:
I have modified the code. Same outcome, different approach, I just excluded the the property "processing_job_ids" from the output via a Flag. See: https://github.com/OCR-D/core/pull/1108/commits

Reply:
Yes, that's better.